packages/cita-signer/src/unsigner.ts

```diff
@@ -1,11 +1,12 @@
-import citaRSA from './config'
+const { publicKeyToAddressSM2 } = require('../../cita-sdk/lib/utils/sm2Utils')
```

Comment:
Does it refer to a `js` file?

Reply:
yes, so, that will be a problem?

Reply:
I was just confused.

Now the `cita-signer` imports `cita-sdk` as its dependency while `cita-signer` is a dependency of `cita-sdk`. And the reference is not achieved by `npm package` but directly by files.

Reply:
these are 2 packages,  `cita-signer` should not know there is a dir like `../../cita-sdk`
this should be fixed.